pyproject.toml

```diff
@@ -25,6 +25,15 @@ packages = [
 ]
 repository = "https://github.com/chanzuckerberg/cryoet-data-portal-neuroglancer"
 
+[tool.poetry]
+name = "cryoet-data-portal-neuroglancer"
+version = "1.1.0"
+description = "Utility package for working with Neuroglancer data in the CZI Cryo-ET Data Portal"
+authors = [
+    "CZI Imaging Team <cryoetdataportal@chanzuckerberg.com>",
+    "Sean Martin",
+    "Vincent Aranega",
+]
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
```

Comment:
Does this actually upgrade poetry?

Reply:
I don't think this spec upgrades or install the poetry requirement.